src/colour_output.rs

```diff
@@ -0,0 +1,37 @@
+use std::str::FromStr;
+
+use palette::Srgb;
+
+pub enum ColourOutput {
+    Hex,
+    Rgb255,
+    Rgb01,
+}
+
+impl ColourOutput {
+    pub fn format_colour(&self, colour: &Srgb<u8>) -> String {
+        match self {
+            Self::Hex => format!("#{:02x}{:02x}{:02x}", colour.red, colour.green, colour.blue),
+            Self::Rgb255 => format!("rgb({}, {}, {})", colour.red, colour.green, colour.blue),
+            Self::Rgb01 => format!(
+                "rgb({:.3}, {:.3}, {:.3})",
+                colour.red as f32 / 255.0,
+                colour.green as f32 / 255.0,
+                colour.blue as f32 / 255.0
+            ),
```

Comment:
I see why you've wrapped these in `rgb(…)`, but that's not what I want. These outputs are meant to be something that can be used by another tool, so I want them deliberately simple so they can be easily parsed.

Compare:

```
rgb(201, 10, 1)
201, 10, 1
```

For the first output, I need to strip away several characters, then split on commas, then cast to ints.

For the second output, I split on commas, then cast to ints.

The second output is more what I had in mind.